instrumentation/net/http/otelhttp/handler.go

```diff
@@ -60,11 +54,16 @@ func defaultHandlerFormatter(operation string, _ *http.Request) string {
 	return operation
 }
 
-// NewHandler wraps the passed handler, functioning like middleware, in a span
-// named after the operation and with any provided Options.
+// NewHandler wraps the passed handler in a span named after the operation and
+// with any provided Options.
 func NewHandler(handler http.Handler, operation string, opts ...Option) http.Handler {
-	h := Handler{
-		handler:   handler,
+	return NewMiddleware(operation, opts...)(handler)
+}
+
+// / NewMiddleware returns a tracing middleware from the given operation name and
+// options.
```

Comment:
It is not only about tracing but also about metrics. Also there is an unnecessary `/ `

Reply:
comment adjusted